### DISCUSSION TITLE
Made some videos where I used ppInk

### DISCUSSION THREAD

**Discussion by rexelbartolome** [pubpub-zz/ppInk#157] [Announcements]
Maybe some snippets/clips of these can be used for some sort of demo video for ppInk 😄

Krita - Digital Painting Tutorials
[![EASY and SIMPLE Way to Shade Your Drawings in Krita - Digital Painting Tutorial](https://img.youtube.com/vi/_DwWmdQ3oLw/0.jpg)](https://www.youtube.com/watch?v=_DwWmdQ3oLw)

[![How to Make Custom Brushes in Krita 5 - Digital Painting Tutorial](https://img.youtube.com/vi/oI7_4FFkBaQ/0.jpg)](https://www.youtube.com/watch?v=oI7_4FFkBaQ)

A breakdown video of a bunch of screenshots from a Netflix series
[![Concept Artist Reacts to Arcane (2021) + Breakdown](https://img.youtube.com/vi/sE99ymm_jEY/0.jpg)](https://www.youtube.com/watch?v=sE99ymm_jEY)

**Comment by pubpub-zz** [comment]
great, 
however my heart is really swinging between ppink demonstration and your great demonstration of krita 😊
